ast/builtins.go

```diff
@@ -2327,6 +2328,41 @@ with all others being treated as intermediates.`,
 	),
 }
 
+var CryptoX509ParseAndVerifyCertificatesWithOptions = &Builtin{
+	Name: "crypto.x509.parse_and_verify_certificates_with_options",
+	Description: `Returns one or more certificates from the given string containing PEM
```

Comment:
Sorry about the back and forth on this. We can update the `Description` text part to:

```
Returns one or more certificates from the given string containing PEM or base64 encoded DER certificates after verifying the supplied certificates form a complete certificate chain back to a trusted root. A config option passed as the second argument can be used to configure the validation options used.

The first certificate is treated as the root and the last is treated as the leaf, with all others being treated as intermediates.

`options` object supports four fields which maps to same fields in [x509.VerifyOptions struct](https://pkg.go.dev/crypto/x509#VerifyOptions) namely `DNSName`, `CurrentTime`, `KeyUsages` and `MaxConstraintComparisions`.  
```

Reply:
In our options, 3 fields are kind of diff types than that in Go lib
`CurrentTime`: number
`KeyUsages`: list of strings,  Go lib ; list of int
`MaxConstraintComparisions`: the types are same but in Go lib its mis-spelled (.. `sions`) extra "i" before "o". Ours is `MaxConstraintComparisons`. Thats why I explained at least these 3 fields. if we don't explain them , user might get confused.

Reply:
Ok thanks for the clarification. We can keep it then.